src/invidious/helpers/serialized_yt_data.cr

```diff
@@ -274,4 +274,62 @@ struct Continuation
   end
 end
 
+struct HashtagPage
```

Comment:
Not sure if these structs should be moved elsewhere?

Reply:
In the future, that could be a good idea. this file (as well as the extractors one) start to be quite clogged up!